crates/algo_fetch/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "algo_fetch"
+version = "0.0.1"
+authors = ["contact@algorand.com"]
```

Comment:
Is there a way to omit/change this when generating? It also shows up in the README

Reply:
We have a few options, `sed` is a good fallback but in this case we may just want to maintain our own OAS for a while. Maybe we can present both at once?

Reply:
Yeah having a seperately maintained OAS for now seems like it makes the most sense. Long term we can plan on PRing upstream but we can quickly iterate if we just have our own OAS short term